### ISSUE TITLE
Ensure unix socket support

### ISSUE THREAD

**Issue by mwalbeck** [MetaProvide/talked#26]

**Comment by mwalbeck** [comment]
@Kixunil I've had a look at the Unix socket support, and thankfully it's looking fantastic. The planned way to handle it with the Talked client would be to use the scheme specified in the URI to determine how to connect to the Talked server. So if the URI starts with `unix://` the client would attempt to connect to a Unix socket at the specified path. This should work in theory as support for Unix sockets was added to PHP curl in PHP 7.0.7, and it works great using straight curl. I've created an issue on the client repo for this https://github.com/MetaProvide/talked-client/issues/9

The internal server supports both TCP and Unix sockets, and it can be configured through the `--host` argument I added recently. To configure Talked to listen to a Unix socket with the internal server simply prefix the desired path with `unix://` like so:
```
talked --host unix:///tmp/talked.sock
```

It's also very simple to configure for uWSGI, simply use --http-socket and provide it the desired path. For example:
```
uwsgi --http-socket /tmp/talked.sock --master --manage-script-name --mount /=talked.main:app
```
So I don't think there is more to do on this on the server side.
